gcc/testsuite/rust/compile/issue-2650.rs

```diff
@@ -0,0 +1,3 @@
+pub async fn a() -> u32 {
+    1
+}
```

Comment:
so this test should eventually fail, unless we are in an edition >= 2018. so please do the following:
```suggestion
// { dg-additional-options "-frust-edition=2018" }

pub async fn a() -> u32 {
    1
}
```

Reply:
I see, I'll do that. I am not familiar with dejagnu , but wouldn't it be "-frust-edition>=2018" or does it handle it on its own?

Reply:
This dejagnu directive pass the argument to the compiler, this means the test code will be compiled using gccrs with the flag `-frust-edition=2018` hence why we're not setting `>=`, because we're setting the exact rust edition, we're not checking against all editions greater than 2018.

Reply:
Oh , I got it now, thanks for the help :D